src/features/user/TreeMapper/components/PlantLocationPage.tsx

```diff
@@ -122,30 +122,48 @@ export function LocationDetails({
             </div> */}
         {location.measurements && (
           <>
-            <div className={styles.singleDetail}>
-              <p className={styles.title}>{t('height')}</p>
-              <div className={styles.value}>
-                {location.measurements?.height}
-                {t('m')}
+            <div className={styles.measurements}>
+              <div className={styles.singleDetail}>
+                <p className={styles.title}>Measurements</p>
```

Comment:
Please do not forgot translations. If this PR is still a draft use "convert to draft" to make it a draft.